packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -532,13 +533,36 @@ class _ForceImplicitScrollPhysics extends ScrollPhysics {
 ///  * [ScrollPhysics], the base class which defines the API for scrolling
 ///    physics.
 ///  * [PageView.physics], which can override the physics used by a page view.
+///  * [PageScrollSimulation], which implements Android page view scroll physics, and
+///    used by this class.
 class PageScrollPhysics extends ScrollPhysics {
   /// Creates physics for a [PageView].
-  const PageScrollPhysics({ super.parent });
+  const PageScrollPhysics({ super.parent, this.useNewPhysics = false });
+
+  /// An opt-in flag for a new implementation of physics.
+  ///
+  /// Enabling this will use scroll physics which are more close to native
+  /// Android and iOS implementations.
+  final bool useNewPhysics;
```

Comment:
How should customers migrate? Do they need to provide a custom scroll physics to every Tab or Page view? How do we migrate them off of it?